### PR TITLE
Complete the EditorSettings class documentation

### PR DESCRIPTION
**Pull request by Calinou**
EditorSettings is now 100% documented.

### CODE REVIEW

File: doc/classes/EditorSettings.xml [hunk 1]
```diff
@@ -174,19 +186,20 @@
...
 			<description>
-				Emitted when editor settings change.
+				Emitted after any editor setting has changed.
 			</description>
 		</signal>
 	</signals>
 	<constants>
 		<constant name="NOTIFICATION_EDITOR_SETTINGS_CHANGED" value="10000">
-			Emitted when editor settings change. It used by various editor plugins to update their visuals on theme changes or logic on configuration changes.
+			Emitted after any editor setting has changed. It used by various editor plugins to update their visuals on theme changes or logic on configuration changes.
```
Comment (VedatGunel) [inline doc/classes/EditorSettings.xml:202]:
```suggestion
			Emitted after any editor setting has changed. It's used by various editor plugins to update their visuals on theme changes or logic on configuration changes.
```

Comment (Calinou) [reply doc/classes/EditorSettings.xml:202]: Good catch :slightly_smiling_face: Thanks!

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.3.

**Pull request merged**